e2e/visual_regression/summary/summary_page.test.ts

```diff
@@ -0,0 +1,43 @@
+import { loginAsVisualRegressionAdmin } from '../../utils/test-util';
+import { expect, test } from '@playwright/test';
+
+// Skip this test because this page will be replaced with a NEO design
+test.describe('Summary page Visual Regression Test', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.setViewportSize({
+      width: 1500,
+      height: 1200,
+    });
+    await loginAsVisualRegressionAdmin(page);
+  });
```

Comment:
we dont need to log in and go to summary page everytime.
```suggestion
 test.beforeEach(async ({ page }) => {
    await page.setViewportSize({
      width: 1500,
      height: 1200,
    });
    await loginAsVisualRegressionAdmin(page);
  });
test.describe('Summary page Visual Regression Test', () => {
 
```